### PR TITLE
fix: updates per pr review

### PR DESCRIPTION
**Pull request by YazeedLoonat**
This is a bit of suggested changes from reviewing: https://github.com/CityOfDetroit/bloom/pull/1181/files?diff=split&w=1

Lemme know what you think

### CODE REVIEW

**Comment by emilyjablonski** [comment]
This LGTM :) Will let @jaredcwhite give feedback

File: sites/public/src/RenderIf.tsx [hunk 1]
```diff
@@ -3,15 +3,14 @@ import { useRouter } from "next/router"
 const RenderIf = (props: { language: string; children: JSX.Element }) => {
   const router = useRouter()
 
-  if (props.language == "all") {
+  if (
```
Comment (jaredcwhite) [inline sites/public/src/RenderIf.tsx:6]: @YazeedLoonat Nice :)

**Review by jaredcwhite — CHANGES_REQUESTED** [review]
@YazeedLoonat Thanks for taking a look at this and the `RenderIf` cleanup.

Switching from a Markdown file to a tsx file puts us out of step with the same sort of page in a variety of other contexts (lots of these various [content pages in Alameda for example](https://github.com/housingbayarea/bloom/tree/0.3_alameda/sites/public/page_content)), and it's also no longer possible to edit in a Markdown editor/syntax highlighting/etc. Also worth noting this only works as-is because `InfoCard` itself has a Markdown processing function…otherwise we'd be loosing all the Markdown within the cards.

So I think it's best to keep as a dedicated Markdown file. Perhaps in the future this will come separately from a CMS.

**Comment by emilyjablonski** [comment]
@jaredcwhite I don't think I agree with that approach, to me it seems like transitioning them all out of `md` files would be preferable so that we get type safety if anything in the component interfaces change. We also aren't currently editing anything in markdown to my knowledge. Happy to check in offline as well!

**Comment by jaredcwhite** [comment]
@emilyjablonski yeah let's touch base on this. TY

**Comment by seanmalbert** [comment]
@emilyjablonski , @jaredcwhite and @YazeedLoonat, what if we did a hybrid solution, that allows us to have the benefits of keeping md files, but also gives us the benefit of having AdditionalResources as a component?

With the way this branch has it, I would say I would prefer md files, because we lost the benefits of using markdown (the formatting), which I'm willing to bet will come in handy at some point as it has in Alameda. However, if we move the copy into md files and import them into AdditionalResources, we could get the best of both worlds, albeit with a little more overhead.

With the way the md files occur in Alameda, I don't think we would want that whole piece available in the content editor, not without some work for people to select components like InfoCard. I can more easily see us devise a way for us to allow the user to just enter in the markdown copy for pages like this, like a repeater that has a title, href and markdown to add in each info card.

Instead of spinning off another branch, I'm just going to include screenshots of a working example.

See here that the top info card is using the imported `test.md`:

Screen Shot 2022-04-29 at 12 55 14 PM

And here is the contents of `test.md`:
Screen Shot 2022-04-29 at 12 55 55 PM

And here it is rendered on the page:
Screen Shot 2022-04-29 at 12 56 48 PM

**Comment by jaredcwhite** [comment]
@seanmalbert I like thinking about how we could structure this in terms of content coming in from elsewhere (maybe a CMS). Is it premature though to structure this without knowing if we'd even continue using Markdown formatting for CMS-y content on the platform?

BTW I made a few changes to the original PR, wondering if you'd seen that yet: https://github.com/CityOfDetroit/bloom/pull/1181

**Comment by seanmalbert** [comment]
> @seanmalbert I like thinking about how we could structure this in terms of content coming in from elsewhere (maybe a CMS). Is it premature though to structure this without knowing if we'd even continue using Markdown formatting for CMS-y content on the platform?
> 
> BTW I made a few changes to the original PR, wondering if you'd seen that yet: #1181

@jaredcwhite ,
Yea, I've been thinking about that too, especially since we recently added what to expect content that contains HTML, which prompted me to open [this issue](https://github.com/CityOfDetroit/bloom/issues/1179) (should move to core). I think most modern editors support Markdown formatting as an option, if we wanted to continue to support it, and if we don't there are plenty of packages out there we can use to convert Markdown to HTML. Either way `InfoCard` can be updated to support parsing raw HTML and the approach could still work.

Since either direction isn't too far away from the other, I don't think it's that important which way we go at this stage, however this feels like a step in the direction we want to go. One more thing I'd like to add, is that one of the benefits Detroit has given us, by being completely separate from HBA, is that we've been able to experiment a little in how we approach things (some good has come out of it like unit groups), so while long term we want to have consistent solutions across all of the projects, we can take this opportunity to test some things out.

**Comment by YazeedLoonat** [comment]
Hey @seanmalbert I think the hybrid solution would be good, we may also want to use it for the case of translations. As far as I can tell right now we don't translate what that markdown text displays. Using the hybrid solution we could have different md files for each language and we can just switch between them as needed

**Comment by seanmalbert** [comment]
> Hey @seanmalbert @jaredcwhite @emilyjablonski I think the hybrid solution would be good, we may also want to use it for the case of translations. As far as I can tell right now we don't translate what that markdown text displays. Using the hybrid solution we could have different md files for each language and we can just switch between them as needed

@YazeedLoonat ,
Multiple translations is supported by the current md solution, fyi. I am going to make the call though and go with what's in this PR, so we can weigh the pros/cons of this workflow against what's in Alameda in a real world scenario.

**Comment by jaredcwhite** [comment]
@seanmalbert @YazeedLoonat All right, in that case it probably makes sense to change this PR to merge directly into dev and I can just close #1181. Sound good?

**Comment by jaredcwhite** [comment]
@seanmalbert @YazeedLoonat OK I'll work on the proposed changes (intend to update the code in PR #1181). I'll keep this open for reference until we need to close it out.

**Comment by YazeedLoonat** [comment]
closing as no longer needed

**Pull request closed without merging**